table/block_based/block_based_table_reader.h

```diff
@@ -273,7 +281,8 @@ class BlockBasedTable : public TableReader {
       FilePrefetchBuffer* prefetch_buffer, const ReadOptions& ro,
       const BlockHandle& handle, const UncompressionDict& uncompression_dict,
       CachableEntry<Block>* block_entry, BlockType block_type,
-      GetContext* get_context, BlockCacheLookupContext* lookup_context) const;
+      GetContext* get_context, BlockCacheLookupContext* lookup_context,
+      BlockContents* contents = nullptr) const;
```

Comment:
If we remove `= nullptr`, will the PR grow much bigger in size? If not, I am for removing default argument value.

Reply:
Shouldn't increase the PR size by much. Will make it non-optional.